src/Refactoring2-Transformations-Tests/RBCopyPackageParametrizedTest.class.st

```diff
@@ -16,59 +16,102 @@ RBCopyPackageParametrizedTest >> constructor [
 	^ #copyPackage:in:
 ]
 
-{ #category : #'failure tests' }
-RBCopyPackageParametrizedTest >> testBadName [
-	self shouldFail: (self createRefactoringWithArguments: 
-		{#'Refactoring-Tests-Core' . #'Refactoring-Tests-Core'}).
+{ #category : #accessing }
+RBCopyPackageParametrizedTest >> createPackageNamed: packageName withClasses: aCollection [
+
+	[ RPackageOrganizer default createPackageNamed: packageName ]
+	on: RPackageConflictError
+	do: [ ].
+
+	aCollection do: [ : className |
+		| refactoring |
+		refactoring := RBInsertClassTransformation
+						addClass: className
+						superclass: #Object
+						subclasses: #()
+						category: packageName.
+		refactoring asRefactoring execute ]
 ]
 
-{ #category : #tests }
-RBCopyPackageParametrizedTest >> testCopyPackage [
-	| refactoring aModel |
-	refactoring := (self createRefactoringWithArguments: 
-		{ #'Refactoring-Tests-Changes' . #'Refactoring-Tests-Changes1'}).
-	aModel := refactoring model.
-	
-	self executeRefactoring: refactoring.
-	
-	self assert: (aModel packageNamed: #'Refactoring-Tests-Changes1') isNotNil.
-	self assert: (aModel classNamed: #RBRefactoringChangeTestCopy) category equals: #'Refactoring-Tests-Changes1'.
-	self assert: (aModel classNamed: #RBRefactoringChangeTestCopy) instanceVariableNames asArray
-		equals: #( 'changes' 'workingEnvironment' 'changeFactory').
-	self assert: (aModel classNamed: #RBRefactoringChangeManagerTestCopy) category equals: #'Refactoring-Tests-Changes1'.
+{ #category : #accessing }
+RBCopyPackageParametrizedTest >> removePackageNamed: packageName [
+
+	RPackageOrganizer default removePackageNamed: packageName
+]
+
+{ #category : #running }
+RBCopyPackageParametrizedTest >> setUp [
+	super setUp.
+	model := self defaultNamespace new.
+	self
+		createPackageNamed: 'RefactoringSmallPackageForTest'
+		withClasses: #('RBFTTest1' 'RBFTTest2')
+]
+
+{ #category : #running }
+RBCopyPackageParametrizedTest >> tearDown [
+
+	super tearDown.
```

Comment:
super tearDown should be the last expression.

Reply:
Ok, fix in https://github.com/pharo-project/pharo/commit/e2dc9b8466c9745af2f71b97a859c170334b27fb